apps/web/next.config.js

```diff
@@ -4,8 +4,19 @@ const os = require("os");
 const englishTranslation = require("./public/static/locales/en/common.json");
 const { withAxiom } = require("next-axiom");
 const { i18n } = require("./next-i18next.config");
-const { pages } = require("./pages");
-const { getSubdomainRegExp } = require("./getSubdomainRegExp");
+const {
+  userTypeRoutePath,
+  teamTypeRoutePath,
+  privateLinkRoutePath,
+  embedUserTypeRoutePath,
+  embedTeamTypeRoutePath,
+  orgHostPath,
+  orgUserRoutePath,
+  orgUserTypeRoutePath,
+  orgUserTypeEmbedRoutePath,
+  otherNonExistingRouteRewrites,
+  rewritesThatConflictWithOrgRewrites,
```

Comment:
Now, all rewrite paths are coming from `pagesAndRewritePaths` so that they can be directly tested. No hardcoding needed.